diff --git a/includes/glad/CMakeLists.txt b/includes/glad/CMakeLists.txt
index 36de7e0..fe5ad43 100644
--- a/includes/glad/CMakeLists.txt
+++ b/includes/glad/CMakeLists.txt
@@ -13,3 +13,5 @@ set(HEADERS
 include_directories(include)
 
 add_library(glad STATIC ${SOURCES} ${HEADERS})
+# Linux, Python module, needs -fPIC and friends to link
+set_property(TARGET glad PROPERTY POSITION_INDEPENDENT_CODE ON)
\ No newline at end of file

